### DISCUSSION TITLE
The domain <domain> is already owned by another Vercel account.

### DISCUSSION THREAD

**Discussion by Jekiwijaya** [vercel/vercel#4654] [Help] [answered]
Hello,

Please I need help, I want to add my domain, I have the domain and use it in firebase hosting right now and want to migrate to my account. But when I want to the domain to my vercel account, it says that the domain is already connected to some vercel account. But none of us using it
> The domain [domain] is already owned by another Vercel account.

How do we can take over?

Please help, thank you

**Comment by styfle** [comment, marked as answer]
Hi @Jekiwijaya 

We'll need some more information to investigate the problem.

Please create a [support request](https://vercel.com/support/request) with your domain details and we'll get it sorted out.

Thanks!
